### PR TITLE
LL-377: Login View

### PR DESCRIPTION
**Pull request by divyanshjain9**

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import IdentitiesPage from './components/IdentitiesPage.jsx';
 import ClaimsPage from './components/ClaimsPage.jsx';
 
 import BlockchainService from './services/BlockchainService.js';
+import Login from './components/Login.jsx';
```
Comment (jkrinsky) [inline ui/src/App.tsx]: the casing for the login filename is wrong--should be Login.jsx, not login.jsx

**Review by jkrinsky — CHANGES_REQUESTED** [review]
just one issue I am seeing with the lower/upper case filename

File: ui/src/App.tsx (hunk 1 above)
Comment (divyanshjain9) [reply ui/src/App.tsx]: @jkrinsky it's updated, you can review now.

**Review by jkrinsky — APPROVED** [review]
LGTM!

**Pull request merged**